src/index.js

```diff
@@ -14,6 +14,7 @@ export { ServiceLogo } from './ServiceLogo';
 export { Stepper } from './Stepper';
 export { default as Text } from './Text';
 export { TextInput } from './TextInput';
+export { Tooltip, TooltipBubble } from './Tooltip';
```

Comment:
Is TooltipBubble necessary to export? It seems to me it is an internal component.

Reply:
Ok, makes sense.